pwem/__init__.py

```diff
@@ -33,11 +33,9 @@
 from protocol import *
 from convert import *
 from wizards import *
-from pyworkflow.utils import importFromPlugin
-from pyworkflow.wizard import Wizard
 from pyworkflow.viewer import Viewer
-from pyworkflow import Config
-
+from pyworkflow.wizard import Wizard
+from filehandlers import *
```

Comment:
Maybe filehandlers should be organized into an existing submodule?
In Py3 you need to make relative import relative: e.g:
```python
from .filehandlers import *
```
Finally, import * are not a good idea anyway.